MekHQ/src/mekhq/campaign/finances/enums/FinancialYearDuration.java

```diff
@@ -71,8 +75,38 @@ public boolean isEndOfFinancialYear(LocalDate today) {
         }
     }
 
+    /**
+     * @param tomorrow tomorrow's date. Because of how this is passed in, we are provided that
+     *                 instead of today
+     * @return the filename to use for the date during financial export
+     */
+    public String getExportFilenameDateString(final LocalDate tomorrow) {
+        final int year = tomorrow.getYear() - 1;
```

Comment:
There is a lot of subtractions from year that I don't follow here, especially given the date is called "tomorrow".

So if I passed in 2021-03-14 (tomorrow), year would be 2020 and then "biannual" would be 2019-2020? Biannual means twice a year, at least where I come from, so I guess I don't follow why its two years and not two dates within the same year.

Reply:
When we hit the export, it will always be after we advance the day. Thus, the day passed in would be the new day, not the actual end of month.

So, how this works is you'll get the export as 2950-2959 when the provided date is 01-JAN-2960 and have set to DECENNIAL, instead of the previous export date of 2960... which makes no sense for finances exporting at the end of the decade. I agree it's confusing, but... it looks nice on the export.

Reply:
BIANNUAL can mean either half a year or once every two years, whereas semiannual is always half a year. It is also specifically mentioned externally in the resource strings.

Further, this can only be hit when the previous is true... so you'll always know the value is the first day of the next year.

Reply:
Biennial is every other year and should be used if that's what we mean. Biennial is unambiguous in that context. Semiannual and biannual are interchangeable.

Reply:
Swapped it over and added Unit Testing.